python/lsst/ip/isr/isrTask.py

```diff
@@ -426,6 +431,8 @@ def run(self, ccdExposure, bias=None, linearizer=None, dark=None, flat=None, def
 
         if self.config.doAssembleCcd:
             ccdExposure = self.assembleCcd.assembleCcd(ccdExposure)
+            if not ccdExposure.getWcs() and self.config.expectWcs:
+                self.log.warn("No WCS found in input exposure")
```

Comment:
There isn't enough context, but if you run the task with `--longlog` you'll get what you need.